tests/test_cli.py

```diff
@@ -20,6 +25,51 @@ def context():
     return ctx().obj["context"]
 
 
+mock_gui_main = """from .app import show
```

Comment:
You can call a module as-is.

You can't call a package as-is, which is where `__main__.py` comes in. It makes a package executable.

Simply move everything into a single file, call it `sys.executable, "-m", "mock_gui"` and it should work.